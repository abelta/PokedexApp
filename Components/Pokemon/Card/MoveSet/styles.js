import { StyleSheet } from 'react-native';

const styles = StyleSheet.create({
  moveSet: {
    flexDirection: 'column',
  },
  title: {
    backgroundColor: 'black',
    color: 'white',
    padding: 2,
    paddingLeft: 10,
  },
  list: {
    flex: 1,
  },
  move: {
    paddingLeft: 10,
  },
});

export default styles;
