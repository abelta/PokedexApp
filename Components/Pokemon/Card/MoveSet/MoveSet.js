import React from 'react';
import PropTypes from 'prop-types';
import { ScrollView, Text, View } from 'react-native';
import { pokemonMoveShape } from '../../../../PropTypes/Pokemon';
import styles from './styles';

const MoveSet = ({ data, style }) => (
  <View style={[styles.moveSet, style]}>
    <Text style={styles.title}>Moves</Text>
    <ScrollView style={styles.list}>
      {
        data.map(move => (
          <Text key={move.move.name} style={styles.move}>{move.move.name}</Text>
        ))
      }
    </ScrollView>
  </View>
);

MoveSet.propTypes = {
  data: PropTypes.arrayOf(pokemonMoveShape).isRequired,
  // eslint-disable-next-line react/require-default-props, react/forbid-prop-types
  style: PropTypes.number,
};

export default MoveSet;
