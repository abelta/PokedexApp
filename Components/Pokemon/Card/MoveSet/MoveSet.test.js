import React from 'react';
import renderer from 'react-test-renderer';
import camelcaseKeys from 'camelcase-keys';
import data from './mock-data.json';
import MoveSet from '.';

describe('MoveSet', () => {
  it('renders correctly', () => {
    const tree = renderer.create(
      <MoveSet data={camelcaseKeys(data, { deep: true }).moves} />,
    ).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
