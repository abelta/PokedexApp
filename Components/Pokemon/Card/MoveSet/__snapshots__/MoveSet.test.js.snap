// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MoveSet renders correctly 1`] = `
<View
  style={
    Array [
      Object {
        "flexDirection": "column",
      },
      undefined,
    ]
  }
>
  <Text
    accessible={true}
    allowFontScaling={true}
    disabled={false}
    ellipsizeMode="tail"
    style={
      Object {
        "backgroundColor": "black",
        "color": "white",
        "padding": 2,
        "paddingLeft": 10,
      }
    }
  >
    Moves
  </Text>
  <RCTScrollView
    style={
      Object {
        "flex": 1,
      }
    }
  >
    <View>
      <Text
        accessible={true}
        allowFontScaling={true}
        disabled={false}
        ellipsizeMode="tail"
        style={
          Object {
            "paddingLeft": 10,
          }
        }
      >
        mega-punch
      </Text>
      <Text
        accessible={true}
        allowFontScaling={true}
        disabled={false}
        ellipsizeMode="tail"
        style={
          Object {
            "paddingLeft": 10,
          }
        }
      >
        fire-punch
      </Text>
      <Text
        accessible={true}
        allowFontScaling={true}
        disabled={false}
        ellipsizeMode="tail"
        style={
          Object {
            "paddingLeft": 10,
          }
        }
      >
        thunder-punch
      </Text>
      <Text
        accessible={true}
        allowFontScaling={true}
        disabled={false}
        ellipsizeMode="tail"
        style={
          Object {
            "paddingLeft": 10,
          }
        }
      >
        scratch
      </Text>
      <Text
        accessible={true}
        allowFontScaling={true}
        disabled={false}
        ellipsizeMode="tail"
        style={
          Object {
            "paddingLeft": 10,
          }
        }
      >
        swords-dance
      </Text>
      <Text
        accessible={true}
        allowFontScaling={true}
        disabled={false}
        ellipsizeMode="tail"
        style={
          Object {
            "paddingLeft": 10,
          }
        }
      >
        cut
      </Text>
      <Text
        accessible={true}
        allowFontScaling={true}
        disabled={false}
        ellipsizeMode="tail"
        style={
          Object {
            "paddingLeft": 10,
          }
        }
      >
        wing-attack
      </Text>
      <Text
        accessible={true}
        allowFontScaling={true}
        disabled={false}
        ellipsizeMode="tail"
        style={
          Object {
            "paddingLeft": 10,
          }
        }
      >
        fly
      </Text>
      <Text
        accessible={true}
        allowFontScaling={true}
        disabled={false}
        ellipsizeMode="tail"
        style={
          Object {
            "paddingLeft": 10,
          }
        }
      >
        mega-kick
      </Text>
      <Text
        accessible={true}
        allowFontScaling={true}
        disabled={false}
        ellipsizeMode="tail"
        style={
          Object {
            "paddingLeft": 10,
          }
        }
      >
        headbutt
      </Text>
      <Text
        accessible={true}
        allowFontScaling={true}
        disabled={false}
        ellipsizeMode="tail"
        style={
          Object {
            "paddingLeft": 10,
          }
        }
      >
        body-slam
      </Text>
      <Text
        accessible={true}
        allowFontScaling={true}
        disabled={false}
        ellipsizeMode="tail"
        style={
          Object {
            "paddingLeft": 10,
          }
        }
      >
        take-down
      </Text>
      <Text
        accessible={true}
        allowFontScaling={true}
        disabled={false}
        ellipsizeMode="tail"
        style={
          Object {
            "paddingLeft": 10,
          }
        }
      >
        double-edge
      </Text>
      <Text
        accessible={true}
        allowFontScaling={true}
        disabled={false}
        ellipsizeMode="tail"
        style={
          Object {
            "paddingLeft": 10,
          }
        }
      >
        leer
      </Text>
      <Text
        accessible={true}
        allowFontScaling={true}
        disabled={false}
        ellipsizeMode="tail"
        style={
          Object {
            "paddingLeft": 10,
          }
        }
      >
        growl
      </Text>
      <Text
        accessible={true}
        allowFontScaling={true}
        disabled={false}
        ellipsizeMode="tail"
        style={
          Object {
            "paddingLeft": 10,
          }
        }
      >
        roar
      </Text>
      <Text
        accessible={true}
        allowFontScaling={true}
        disabled={false}
        ellipsizeMode="tail"
        style={
          Object {
            "paddingLeft": 10,
          }
        }
      >
        ember
      </Text>
      <Text
        accessible={true}
        allowFontScaling={true}
        disabled={false}
        ellipsizeMode="tail"
        style={
          Object {
            "paddingLeft": 10,
          }
        }
      >
        flamethrower
      </Text>
      <Text
        accessible={true}
        allowFontScaling={true}
        disabled={false}
        ellipsizeMode="tail"
        style={
          Object {
            "paddingLeft": 10,
          }
        }
      >
        hyper-beam
      </Text>
      <Text
        accessible={true}
        allowFontScaling={true}
        disabled={false}
        ellipsizeMode="tail"
        style={
          Object {
            "paddingLeft": 10,
          }
        }
      >
        submission
      </Text>
      <Text
        accessible={true}
        allowFontScaling={true}
        disabled={false}
        ellipsizeMode="tail"
        style={
          Object {
            "paddingLeft": 10,
          }
        }
      >
        counter
      </Text>
      <Text
        accessible={true}
        allowFontScaling={true}
        disabled={false}
        ellipsizeMode="tail"
        style={
          Object {
            "paddingLeft": 10,
          }
        }
      >
        seismic-toss
      </Text>
      <Text
        accessible={true}
        allowFontScaling={true}
        disabled={false}
        ellipsizeMode="tail"
        style={
          Object {
            "paddingLeft": 10,
          }
        }
      >
        strength
      </Text>
      <Text
        accessible={true}
        allowFontScaling={true}
        disabled={false}
        ellipsizeMode="tail"
        style={
          Object {
            "paddingLeft": 10,
          }
        }
      >
        solar-beam
      </Text>
      <Text
        accessible={true}
        allowFontScaling={true}
        disabled={false}
        ellipsizeMode="tail"
        style={
          Object {
            "paddingLeft": 10,
          }
        }
      >
        dragon-rage
      </Text>
      <Text
        accessible={true}
        allowFontScaling={true}
        disabled={false}
        ellipsizeMode="tail"
        style={
          Object {
            "paddingLeft": 10,
          }
        }
      >
        fire-spin
      </Text>
      <Text
        accessible={true}
        allowFontScaling={true}
        disabled={false}
        ellipsizeMode="tail"
        style={
          Object {
            "paddingLeft": 10,
          }
        }
      >
        earthquake
      </Text>
      <Text
        accessible={true}
        allowFontScaling={true}
        disabled={false}
        ellipsizeMode="tail"
        style={
          Object {
            "paddingLeft": 10,
          }
        }
      >
        fissure
      </Text>
      <Text
        accessible={true}
        allowFontScaling={true}
        disabled={false}
        ellipsizeMode="tail"
        style={
          Object {
            "paddingLeft": 10,
          }
        }
      >
        dig
      </Text>
      <Text
        accessible={true}
        allowFontScaling={true}
        disabled={false}
        ellipsizeMode="tail"
        style={
          Object {
            "paddingLeft": 10,
          }
        }
      >
        toxic
      </Text>
      <Text
        accessible={true}
        allowFontScaling={true}
        disabled={false}
        ellipsizeMode="tail"
        style={
          Object {
            "paddingLeft": 10,
          }
        }
      >
        rage
      </Text>
      <Text
        accessible={true}
        allowFontScaling={true}
        disabled={false}
        ellipsizeMode="tail"
        style={
          Object {
            "paddingLeft": 10,
          }
        }
      >
        mimic
      </Text>
      <Text
        accessible={true}
        allowFontScaling={true}
        disabled={false}
        ellipsizeMode="tail"
        style={
          Object {
            "paddingLeft": 10,
          }
        }
      >
        double-team
      </Text>
      <Text
        accessible={true}
        allowFontScaling={true}
        disabled={false}
        ellipsizeMode="tail"
        style={
          Object {
            "paddingLeft": 10,
          }
        }
      >
        smokescreen
      </Text>
      <Text
        accessible={true}
        allowFontScaling={true}
        disabled={false}
        ellipsizeMode="tail"
        style={
          Object {
            "paddingLeft": 10,
          }
        }
      >
        defense-curl
      </Text>
      <Text
        accessible={true}
        allowFontScaling={true}
        disabled={false}
        ellipsizeMode="tail"
        style={
          Object {
            "paddingLeft": 10,
          }
        }
      >
        reflect
      </Text>
      <Text
        accessible={true}
        allowFontScaling={true}
        disabled={false}
        ellipsizeMode="tail"
        style={
          Object {
            "paddingLeft": 10,
          }
        }
      >
        bide
      </Text>
      <Text
        accessible={true}
        allowFontScaling={true}
        disabled={false}
        ellipsizeMode="tail"
        style={
          Object {
            "paddingLeft": 10,
          }
        }
      >
        fire-blast
      </Text>
      <Text
        accessible={true}
        allowFontScaling={true}
        disabled={false}
        ellipsizeMode="tail"
        style={
          Object {
            "paddingLeft": 10,
          }
        }
      >
        swift
      </Text>
      <Text
        accessible={true}
        allowFontScaling={true}
        disabled={false}
        ellipsizeMode="tail"
        style={
          Object {
            "paddingLeft": 10,
          }
        }
      >
        skull-bash
      </Text>
      <Text
        accessible={true}
        allowFontScaling={true}
        disabled={false}
        ellipsizeMode="tail"
        style={
          Object {
            "paddingLeft": 10,
          }
        }
      >
        rest
      </Text>
      <Text
        accessible={true}
        allowFontScaling={true}
        disabled={false}
        ellipsizeMode="tail"
        style={
          Object {
            "paddingLeft": 10,
          }
        }
      >
        rock-slide
      </Text>
      <Text
        accessible={true}
        allowFontScaling={true}
        disabled={false}
        ellipsizeMode="tail"
        style={
          Object {
            "paddingLeft": 10,
          }
        }
      >
        slash
      </Text>
      <Text
        accessible={true}
        allowFontScaling={true}
        disabled={false}
        ellipsizeMode="tail"
        style={
          Object {
            "paddingLeft": 10,
          }
        }
      >
        substitute
      </Text>
      <Text
        accessible={true}
        allowFontScaling={true}
        disabled={false}
        ellipsizeMode="tail"
        style={
          Object {
            "paddingLeft": 10,
          }
        }
      >
        snore
      </Text>
      <Text
        accessible={true}
        allowFontScaling={true}
        disabled={false}
        ellipsizeMode="tail"
        style={
          Object {
            "paddingLeft": 10,
          }
        }
      >
        curse
      </Text>
      <Text
        accessible={true}
        allowFontScaling={true}
        disabled={false}
        ellipsizeMode="tail"
        style={
          Object {
            "paddingLeft": 10,
          }
        }
      >
        protect
      </Text>
      <Text
        accessible={true}
        allowFontScaling={true}
        disabled={false}
        ellipsizeMode="tail"
        style={
          Object {
            "paddingLeft": 10,
          }
        }
      >
        scary-face
      </Text>
      <Text
        accessible={true}
        allowFontScaling={true}
        disabled={false}
        ellipsizeMode="tail"
        style={
          Object {
            "paddingLeft": 10,
          }
        }
      >
        mud-slap
      </Text>
      <Text
        accessible={true}
        allowFontScaling={true}
        disabled={false}
        ellipsizeMode="tail"
        style={
          Object {
            "paddingLeft": 10,
          }
        }
      >
        outrage
      </Text>
      <Text
        accessible={true}
        allowFontScaling={true}
        disabled={false}
        ellipsizeMode="tail"
        style={
          Object {
            "paddingLeft": 10,
          }
        }
      >
        sandstorm
      </Text>
      <Text
        accessible={true}
        allowFontScaling={true}
        disabled={false}
        ellipsizeMode="tail"
        style={
          Object {
            "paddingLeft": 10,
          }
        }
      >
        endure
      </Text>
      <Text
        accessible={true}
        allowFontScaling={true}
        disabled={false}
        ellipsizeMode="tail"
        style={
          Object {
            "paddingLeft": 10,
          }
        }
      >
        swagger
      </Text>
      <Text
        accessible={true}
        allowFontScaling={true}
        disabled={false}
        ellipsizeMode="tail"
        style={
          Object {
            "paddingLeft": 10,
          }
        }
      >
        fury-cutter
      </Text>
      <Text
        accessible={true}
        allowFontScaling={true}
        disabled={false}
        ellipsizeMode="tail"
        style={
          Object {
            "paddingLeft": 10,
          }
        }
      >
        steel-wing
      </Text>
      <Text
        accessible={true}
        allowFontScaling={true}
        disabled={false}
        ellipsizeMode="tail"
        style={
          Object {
            "paddingLeft": 10,
          }
        }
      >
        attract
      </Text>
      <Text
        accessible={true}
        allowFontScaling={true}
        disabled={false}
        ellipsizeMode="tail"
        style={
          Object {
            "paddingLeft": 10,
          }
        }
      >
        sleep-talk
      </Text>
      <Text
        accessible={true}
        allowFontScaling={true}
        disabled={false}
        ellipsizeMode="tail"
        style={
          Object {
            "paddingLeft": 10,
          }
        }
      >
        return
      </Text>
      <Text
        accessible={true}
        allowFontScaling={true}
        disabled={false}
        ellipsizeMode="tail"
        style={
          Object {
            "paddingLeft": 10,
          }
        }
      >
        frustration
      </Text>
      <Text
        accessible={true}
        allowFontScaling={true}
        disabled={false}
        ellipsizeMode="tail"
        style={
          Object {
            "paddingLeft": 10,
          }
        }
      >
        dynamic-punch
      </Text>
      <Text
        accessible={true}
        allowFontScaling={true}
        disabled={false}
        ellipsizeMode="tail"
        style={
          Object {
            "paddingLeft": 10,
          }
        }
      >
        dragon-breath
      </Text>
      <Text
        accessible={true}
        allowFontScaling={true}
        disabled={false}
        ellipsizeMode="tail"
        style={
          Object {
            "paddingLeft": 10,
          }
        }
      >
        iron-tail
      </Text>
      <Text
        accessible={true}
        allowFontScaling={true}
        disabled={false}
        ellipsizeMode="tail"
        style={
          Object {
            "paddingLeft": 10,
          }
        }
      >
        metal-claw
      </Text>
      <Text
        accessible={true}
        allowFontScaling={true}
        disabled={false}
        ellipsizeMode="tail"
        style={
          Object {
            "paddingLeft": 10,
          }
        }
      >
        hidden-power
      </Text>
      <Text
        accessible={true}
        allowFontScaling={true}
        disabled={false}
        ellipsizeMode="tail"
        style={
          Object {
            "paddingLeft": 10,
          }
        }
      >
        twister
      </Text>
      <Text
        accessible={true}
        allowFontScaling={true}
        disabled={false}
        ellipsizeMode="tail"
        style={
          Object {
            "paddingLeft": 10,
          }
        }
      >
        sunny-day
      </Text>
      <Text
        accessible={true}
        allowFontScaling={true}
        disabled={false}
        ellipsizeMode="tail"
        style={
          Object {
            "paddingLeft": 10,
          }
        }
      >
        rock-smash
      </Text>
      <Text
        accessible={true}
        allowFontScaling={true}
        disabled={false}
        ellipsizeMode="tail"
        style={
          Object {
            "paddingLeft": 10,
          }
        }
      >
        heat-wave
      </Text>
      <Text
        accessible={true}
        allowFontScaling={true}
        disabled={false}
        ellipsizeMode="tail"
        style={
          Object {
            "paddingLeft": 10,
          }
        }
      >
        will-o-wisp
      </Text>
      <Text
        accessible={true}
        allowFontScaling={true}
        disabled={false}
        ellipsizeMode="tail"
        style={
          Object {
            "paddingLeft": 10,
          }
        }
      >
        facade
      </Text>
      <Text
        accessible={true}
        allowFontScaling={true}
        disabled={false}
        ellipsizeMode="tail"
        style={
          Object {
            "paddingLeft": 10,
          }
        }
      >
        focus-punch
      </Text>
      <Text
        accessible={true}
        allowFontScaling={true}
        disabled={false}
        ellipsizeMode="tail"
        style={
          Object {
            "paddingLeft": 10,
          }
        }
      >
        brick-break
      </Text>
      <Text
        accessible={true}
        allowFontScaling={true}
        disabled={false}
        ellipsizeMode="tail"
        style={
          Object {
            "paddingLeft": 10,
          }
        }
      >
        secret-power
      </Text>
      <Text
        accessible={true}
        allowFontScaling={true}
        disabled={false}
        ellipsizeMode="tail"
        style={
          Object {
            "paddingLeft": 10,
          }
        }
      >
        blast-burn
      </Text>
      <Text
        accessible={true}
        allowFontScaling={true}
        disabled={false}
        ellipsizeMode="tail"
        style={
          Object {
            "paddingLeft": 10,
          }
        }
      >
        air-cutter
      </Text>
      <Text
        accessible={true}
        allowFontScaling={true}
        disabled={false}
        ellipsizeMode="tail"
        style={
          Object {
            "paddingLeft": 10,
          }
        }
      >
        overheat
      </Text>
      <Text
        accessible={true}
        allowFontScaling={true}
        disabled={false}
        ellipsizeMode="tail"
        style={
          Object {
            "paddingLeft": 10,
          }
        }
      >
        rock-tomb
      </Text>
      <Text
        accessible={true}
        allowFontScaling={true}
        disabled={false}
        ellipsizeMode="tail"
        style={
          Object {
            "paddingLeft": 10,
          }
        }
      >
        aerial-ace
      </Text>
      <Text
        accessible={true}
        allowFontScaling={true}
        disabled={false}
        ellipsizeMode="tail"
        style={
          Object {
            "paddingLeft": 10,
          }
        }
      >
        dragon-claw
      </Text>
      <Text
        accessible={true}
        allowFontScaling={true}
        disabled={false}
        ellipsizeMode="tail"
        style={
          Object {
            "paddingLeft": 10,
          }
        }
      >
        roost
      </Text>
      <Text
        accessible={true}
        allowFontScaling={true}
        disabled={false}
        ellipsizeMode="tail"
        style={
          Object {
            "paddingLeft": 10,
          }
        }
      >
        natural-gift
      </Text>
      <Text
        accessible={true}
        allowFontScaling={true}
        disabled={false}
        ellipsizeMode="tail"
        style={
          Object {
            "paddingLeft": 10,
          }
        }
      >
        tailwind
      </Text>
      <Text
        accessible={true}
        allowFontScaling={true}
        disabled={false}
        ellipsizeMode="tail"
        style={
          Object {
            "paddingLeft": 10,
          }
        }
      >
        fling
      </Text>
      <Text
        accessible={true}
        allowFontScaling={true}
        disabled={false}
        ellipsizeMode="tail"
        style={
          Object {
            "paddingLeft": 10,
          }
        }
      >
        flare-blitz
      </Text>
      <Text
        accessible={true}
        allowFontScaling={true}
        disabled={false}
        ellipsizeMode="tail"
        style={
          Object {
            "paddingLeft": 10,
          }
        }
      >
        air-slash
      </Text>
      <Text
        accessible={true}
        allowFontScaling={true}
        disabled={false}
        ellipsizeMode="tail"
        style={
          Object {
            "paddingLeft": 10,
          }
        }
      >
        dragon-pulse
      </Text>
      <Text
        accessible={true}
        allowFontScaling={true}
        disabled={false}
        ellipsizeMode="tail"
        style={
          Object {
            "paddingLeft": 10,
          }
        }
      >
        focus-blast
      </Text>
      <Text
        accessible={true}
        allowFontScaling={true}
        disabled={false}
        ellipsizeMode="tail"
        style={
          Object {
            "paddingLeft": 10,
          }
        }
      >
        giga-impact
      </Text>
      <Text
        accessible={true}
        allowFontScaling={true}
        disabled={false}
        ellipsizeMode="tail"
        style={
          Object {
            "paddingLeft": 10,
          }
        }
      >
        shadow-claw
      </Text>
      <Text
        accessible={true}
        allowFontScaling={true}
        disabled={false}
        ellipsizeMode="tail"
        style={
          Object {
            "paddingLeft": 10,
          }
        }
      >
        fire-fang
      </Text>
      <Text
        accessible={true}
        allowFontScaling={true}
        disabled={false}
        ellipsizeMode="tail"
        style={
          Object {
            "paddingLeft": 10,
          }
        }
      >
        defog
      </Text>
      <Text
        accessible={true}
        allowFontScaling={true}
        disabled={false}
        ellipsizeMode="tail"
        style={
          Object {
            "paddingLeft": 10,
          }
        }
      >
        captivate
      </Text>
      <Text
        accessible={true}
        allowFontScaling={true}
        disabled={false}
        ellipsizeMode="tail"
        style={
          Object {
            "paddingLeft": 10,
          }
        }
      >
        ominous-wind
      </Text>
      <Text
        accessible={true}
        allowFontScaling={true}
        disabled={false}
        ellipsizeMode="tail"
        style={
          Object {
            "paddingLeft": 10,
          }
        }
      >
        hone-claws
      </Text>
      <Text
        accessible={true}
        allowFontScaling={true}
        disabled={false}
        ellipsizeMode="tail"
        style={
          Object {
            "paddingLeft": 10,
          }
        }
      >
        flame-burst
      </Text>
      <Text
        accessible={true}
        allowFontScaling={true}
        disabled={false}
        ellipsizeMode="tail"
        style={
          Object {
            "paddingLeft": 10,
          }
        }
      >
        flame-charge
      </Text>
      <Text
        accessible={true}
        allowFontScaling={true}
        disabled={false}
        ellipsizeMode="tail"
        style={
          Object {
            "paddingLeft": 10,
          }
        }
      >
        round
      </Text>
      <Text
        accessible={true}
        allowFontScaling={true}
        disabled={false}
        ellipsizeMode="tail"
        style={
          Object {
            "paddingLeft": 10,
          }
        }
      >
        echoed-voice
      </Text>
      <Text
        accessible={true}
        allowFontScaling={true}
        disabled={false}
        ellipsizeMode="tail"
        style={
          Object {
            "paddingLeft": 10,
          }
        }
      >
        sky-drop
      </Text>
      <Text
        accessible={true}
        allowFontScaling={true}
        disabled={false}
        ellipsizeMode="tail"
        style={
          Object {
            "paddingLeft": 10,
          }
        }
      >
        incinerate
      </Text>
      <Text
        accessible={true}
        allowFontScaling={true}
        disabled={false}
        ellipsizeMode="tail"
        style={
          Object {
            "paddingLeft": 10,
          }
        }
      >
        inferno
      </Text>
      <Text
        accessible={true}
        allowFontScaling={true}
        disabled={false}
        ellipsizeMode="tail"
        style={
          Object {
            "paddingLeft": 10,
          }
        }
      >
        fire-pledge
      </Text>
      <Text
        accessible={true}
        allowFontScaling={true}
        disabled={false}
        ellipsizeMode="tail"
        style={
          Object {
            "paddingLeft": 10,
          }
        }
      >
        bulldoze
      </Text>
      <Text
        accessible={true}
        allowFontScaling={true}
        disabled={false}
        ellipsizeMode="tail"
        style={
          Object {
            "paddingLeft": 10,
          }
        }
      >
        dragon-tail
      </Text>
      <Text
        accessible={true}
        allowFontScaling={true}
        disabled={false}
        ellipsizeMode="tail"
        style={
          Object {
            "paddingLeft": 10,
          }
        }
      >
        confide
      </Text>
      <Text
        accessible={true}
        allowFontScaling={true}
        disabled={false}
        ellipsizeMode="tail"
        style={
          Object {
            "paddingLeft": 10,
          }
        }
      >
        power-up-punch
      </Text>
    </View>
  </RCTScrollView>
</View>
`;
